src/Dfe.PlanTech.Domain/Content/Models/ContentSupport/Heading.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace Dfe.PlanTech.Domain.Content.Models.ContentSupport;
+
+[ExcludeFromCodeCoverage]
+public class CSHeading : ContentBase;
```

Comment:
I think it would be good to have the filename and classname match

Reply:
[856115a](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/900/commits/856115aa950cae32090b514b9ea78f210fd8254b) Good point!